### PR TITLE
Editorial suggestions

### PR DESCRIPTION
**Pull request by mnot**
I believe these are all editorial.

### CODE REVIEW

File: draft-ietf-httpapi-ratelimit-headers.md [hunk 1]
```diff
@@ -144,47 +124,60 @@ The following features are out of the scope of this document:
...
-The "service-limit" is a value associated to the maximum number of requests
-that the server is willing to accept
-from one or more clients
-on a given basis (originating IP, authenticated user, geographical, ..)
-during a [ time window](#time-window).
+## Service Limit {#service-limit}
 
-The service-limit is a non-negative Integer
-expressed in "quota-units".
+The service limit is the maximum number of requests that the server is willing to accept from one or more clients on a given basis (originating IP, authenticated user, geographical, ..) during a [time window](#time-window).
```
Comment (ioggstream) [inline draft-ietf-httpapi-ratelimit-headers.md:174]:
```suggestion
The service limit is associated to the maximum number of requests that the server is willing to accept from one or more clients on a given basis (originating IP, authenticated user, geographical, ..) during a [time window](#time-window).
```

File: draft-ietf-httpapi-ratelimit-headers.md [hunk 2]
```diff
@@ -144,47 +124,60 @@ The following features are out of the scope of this document:
...
 
-The service-limit SHOULD match the maximum number of acceptable requests.
+The service limit is a non-negative Integer expressed in quota units.
 
-The service-limit MAY differ from the total number of acceptable requests
-when weight mechanisms, bursts, or other server policies are implemented.
+The service limit SHOULD match the maximum number of acceptable requests. However, the service limit MAY differ from the total number of acceptable requests when weight mechanisms, bursts, or other server policies are implemented.
```
Comment (ioggstream) [inline draft-ietf-httpapi-ratelimit-headers.md:178]:
```suggestion
The service limit SHOULD match the maximum number of acceptable requests.
However, the service limit MAY differ from the total number of acceptable requests when weight mechanisms, bursts, or other server policies are implemented.
```

File: draft-ietf-httpapi-ratelimit-headers.md [hunk 3]
```diff
@@ -199,383 +193,219 @@ GET /books?author=WuMing ; service-limit=4, remaining: 1, status=200
...
 
-# Providing RateLimit fields {#providing-ratelimit-fields}
-
-A server uses the RateLimit fields to communicate its quota policies
-according to the following rules:
+## RateLimit-Policy {#ratelimit-policy-field}
 
-- RateLimit-Limit and RateLimit-Reset fields are REQUIRED;
-- RateLimit-Remaining field is RECOMMENDED.
+The "RateLimit-Policy" field response field indicates the quota policies currently associated with the client. Its value is informative.
```
Comment (ioggstream) [inline draft-ietf-httpapi-ratelimit-headers.md:222]:
```suggestion
The "RateLimit-Policy"  response field indicates the quota policies currently associated with the client. Its value is informative.
```

**Comment by ioggstream** [comment]
Thanks @mnot! It seems the PR is ok. I partly reverted the wrapping removal since I it really helps me in reviewing and minimizing the diffs but I think the result makes the document more readable.

@unleashed could you please take a look too?

File: draft-ietf-httpapi-ratelimit-headers.md [hunk 4]
```diff
@@ -42,58 +43,37 @@ informative:
...
+This document defines the RateLimit-Limit, RateLimit-Remaining, RateLimit-Reset and RateLimit-Policy HTTP fields for servers to advertise their current service rate limits, thereby allowing clients to avoid being throttled.
 
 --- note_Note_to_Readers
 
 *RFC EDITOR: please remove this section before publication*
 
-Discussion of this draft takes place on the HTTP working group mailing list
-(httpapi@ietf.org), which is archived at
-<https://mailarchive.ietf.org/arch/browse/httpapi/>.
+Discussion of this draft takes place on the HTTP working group mailing list (httpapi@ietf.org), which is archived at <https://mailarchive.ietf.org/arch/browse/httpapi/>.
```
Comment (darrelmiller) [inline draft-ietf-httpapi-ratelimit-headers.md:52]:
```suggestion
Discussion of this draft takes place on the HTTPAPI working group mailing list (httpapi@ietf.org), which is archived at <https://mailarchive.ietf.org/arch/browse/httpapi/>.
```

File: draft-ietf-httpapi-ratelimit-headers.md [hunk 5]
```diff
@@ -602,28 +444,25 @@ RateLimit-Reset: 10
...
+RateLimit fields may contain unexpected values by chance or on purpose.
 For example, an excessively high RateLimit-Remaining field value may be:
 
 - used by a malicious intermediary to trigger a Denial of Service attack
   or consume client resources boosting its requests;
 - passed by a misconfigured server;
 
-or an high RateLimit-Reset field value could inhibit clients to contact
-the server.
+or an high RateLimit-Reset field value could inhibit clients to contact the server.
```
Comment (unleashed) [inline draft-ietf-httpapi-ratelimit-headers.md:465]:
```suggestion
or a high RateLimit-Reset field value could inhibit clients to contact the server.
```

File: draft-ietf-httpapi-ratelimit-headers.md [hunk 6]
```diff
@@ -199,378 +193,227 @@ GET /books?author=WuMing ; service-limit=4, remaining: 1, status=200
...
-In case of resource saturation, the server MAY artificially lower the returned
-values or not serve the request regardless of the advertised quotas.
+In case of resource saturation, the server MAY artificially lower the returned values
+or not serve the request regardless of the advertised quotas.
 
 ## Reliability of RateLimit-Reset {#sec-reset-reliability}
 
-Consider that service-limit might not be restored after the moment referenced by RateLimit-Reset field,
+Consider that service limit might not be restored after the moment referenced by RateLimit-Reset field,
 and the RateLimit-Reset field value do not be considered fixed nor constant.
```
Comment (unleashed) [inline draft-ietf-httpapi-ratelimit-headers.md:408]:
```suggestion
and the RateLimit-Reset field value may not be fixed nor constant.
```

**Review by unleashed — APPROVED** [review]
This is a very nice improvement, thank you!

Commented only on a couple places for typos/minor rephrasing.

**Pull request merged**